docs/api/wanchain.md

```diff
@@ -1,5 +1,255 @@
+
+1. Maker (address creating the delegate deploy) defines the deploy (defines the data for the `AssetLedger` that will be created, who will be the owner of that `AssetLedger` and what amount of tokens will be sent to what address in exchange).
+2. Maker generates the deploy claim and signs it (`claim` functions).
+3. Maker approves value transfer.
+4. Maker sends deploy and signature to the Taker trough arbitrary channel.
+6. Taker performs the deploy.
+
+`Deploy` class is responsible for defining what will happen in the atomic swap. We support two different deploy configurations which we will call a fixed deploy and a dynamic deploy.
+
+In fixed deploy, the Taker of the deploy (its wallet address) is known, and we want to make an atomic deploy specifically with him and only him. For this, we need to set `deploy.takerId`.
+
+In dynamic deploy, we do not care who performs the deploy. In this case, we do not set `deploy.takerId`. Now any account (wallet) will be able to perform such deploy and will automatically become its Taker.
+
+### DeployGateway(provider, deployGatewayId)
+
+A `class` which represents a smart contract on the Wanchain blockchain.
+
+**Arguments**
+
+| Argument | Description
+|-|-
+| deployGatewayId | [required] A `string` representing an address of the [0xcert deploy gateway smart contract](#public-addresses) on the Wanchain blockchain.
+| provider | [required] An instance of an provider.
+
+**Usage**
+
+```ts
+import { HttpProvider } from '@0xcert/wanchain-http-provider';
+import { DeployGateway } from '@0xcert/wanchain-deploy-gateway';
+
+// arbitrary data
+const provider = new HttpProvider();
+const deployGatewayId = '0x073d230a53bffc8295d9a5247296213298e3fbcf';
+
+// create deploy gateway instance
+const deployGateway = new DeployGateway(provider, deployGatewayId);
+```
+
+### cancel(deploy)
+
+An `asynchronous` class instance `function` which marks the provided `deploy` as canceled. This prevents the `deploy` to be performed.
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| deploy.assetLedgerData.capabilities | [required] An array of `AssetLedgerCapability` representing the capabilities of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.name | [required] A `string` representing the name of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.owner | [required] A `string` representing the address of owner of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.schemaId | [required] A `string` representing the schemaId of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.symbol | [required] A `string` representing the symbol of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.uriBase | [required] A `string` representing the uriBase of the `AssetLedger` that will be deployed.
+| deploy.expiration | [required] An `integer` number representing the timestamp in milliseconds at which the deploy expires and can not be performed any more.
```

Comment:
| deploy.expiration | [required] An `integer` number representing the timestamp in milliseconds after which the deploy expires and can not be performed any more.